screens/Main.tsx

```diff
@@ -84,9 +85,10 @@ export default function Main() {
           theme={navigationTheme}
           linking={linking}
           ref={(r) => {
-            if (r) {
-              converseNavigations["main"] = r;
-            }
+            logger.info(
+              `[Main] Setting navigation ref to ${r ? "not null" : "null"}`
+            );
+            setConverseNavigatorRef(r);
```

Comment:
We only have one navigation ref now that we are no longer supporting different layouts and stuff so I did some refactoring to reflect that